conjure-java-core/src/test/resources/example-types-no-static-factory.yml

```
@@ -0,0 +1,20 @@
+types:
+  imports:
+    ExampleExternalReference:
+      base-type: string
+      external:
+        java: test.api.ExampleExternalReference
+  definitions:
+    default-package: com.palantir.product
+    objects:
+      EmptyExampleNoStaticFactory:
```

nit: I think in this case we do have a static factory generated.
```suggestion
      EmptyExample:
```